compiler/rustc_mir/src/transform/check_consts/validation.rs

```diff
@@ -888,15 +888,6 @@ impl Visitor<'tcx> for Validator<'mir, 'tcx> {
 
                 let is_intrinsic = tcx.fn_sig(callee).abi() == RustIntrinsic;
 
-                // HACK: This is to "unstabilize" the `transmute` intrinsic
-                // within const fns. `transmute` is allowed in all other const contexts.
-                // This won't really scale to more intrinsics or functions. Let's allow const
-                // transmutes in const fn before we add more hacks to this.
-                if is_intrinsic && tcx.item_name(callee) == sym::transmute {
-                    self.check_op(ops::Transmute);
```

Comment:
You can also delete the `Transmute ` struct now

Reply:
Will do when I'm able to get tests passing (or passing sans this change). Turns out there's some minor things in the compiler that make it a tad difficult.

Reply:
Is it the 32 bit tests failing issue? Depending on your host OS you can run x.py with `--target=i686_linux_unknown_gnu` or some 32 bit target that fits your host OS. This will automatically build the 32 bit libstd and run the tests you selected on 32 bit. You'll need to do it for both ui tests and mir-opt tests

Reply:
The code I have locally now is slightly different (I think, I literally just got off of it). It's an unrelated test that's preventing me from doing `x t --bless --stage 2`. Something to do with CJK text that is system-specific apparently: #85770.

Most likely I will apply these changes locally, bless the tests, revert that change and commit to see what happens. Just a couple of hiccups along the way, really.